usort/config.py

```diff
@@ -45,6 +52,10 @@ class Config:
     )
     default_section: Category = CAT_THIRD_PARTY
 
+    # Known set of modules with import side effects. These will be implicitly treated
+    # as block separators, similar to non-import statements.
+    side_effect_modules: List[str] = field(default_factory=side_effect_factory)
```

Comment:
I don't have anything close to an exhaustive list across pypi; I'd prefer we default this to an empty sequence, and give a good example of how to include this in config.